src/evolutions/src/main/scala/play/api/db/slick/evolutions/internal/DBApiAdapter.scala

```diff
@@ -49,18 +49,21 @@ private[evolutions] object DBApiAdapter {
       }
     }
     def url: String = dbConfig.db.createSession().metaData.getURL
-    def getConnection(): Connection = {
+    def getConnection(): Connection = getConnection(autocommit = true)
```

Comment:
The reason why I suggested the change is that the default should be the one set by Slick.